targets/thingsplus-rp2040.json

```diff
@@ -0,0 +1,11 @@
+{
+    "inherits": [
+        "rp2040"
+    ],
+    "serial": "uart",
+    "build-tags": ["thingsplus_rp2040"],
+    "linkerscript": "targets/thingsplus-rp2040.ld",
```

Comment:
I think you do not need a separate linker file `targets/thingsplus-rp2040.ld` since it is the same as the pico board you can  specify `targets/pico.ld` here and not add the separate file in this PR.

Reply:
the ld file is different. The thingsplus has 16mb of flash. but that‘s the only diff, i have to admit

Reply:
Oops I missed that difference. In that case, never mind!